test/relations.test.js

```diff
@@ -4092,6 +4092,21 @@ describe('relations', function() {
       });
     });
 
+    it('should destroy all related instances', function(done) {
+      Article.create(function(e, article) {
```

Comment:
nitpick: `if (e) return done(e)`, or you can write it in promisified style to avoid multiple checks for `error`